modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeJavaSetResultsSetTotalMethodCheck.testjava

```diff
@@ -5,32 +5,25 @@
 
 package com.liferay.source.formatter.dependencies.upgrade;
 
-/**
- * @author Tamyris Bernardo
- */
 public class UpgradeJavaSetResultsSetTotalMethodCheck {
 
 	public void method() {
-		SearchContainer<AnnouncementsEntry>
-			announcementsEntriesSearchContainer = new SearchContainer();
+		SearchContainer<AnnouncementsEntry> searchContainer = new SearchContainer();
```

Comment:
@TamyrisTorres, from what I could see in the source code, the model (like `AnnouncementsEntry`) is mandatory in the declaration of a `SearchContainer` variable, in light of this can you please add it to all test files to make all of them consistent?

You may use just `Entry` instead of `AnnouncementsEntry` though it's not a valid model, however it works for the tests purpose. :slightly_smiling_face: